grails-cli/src/main/java/org/grails/forge/cli/util/GrailsVersionProvider.java

```diff
@@ -45,6 +46,7 @@ public class GrailsVersionProvider implements IVersionProvider {
     public String[] getVersion() {
         return new String[] {
                 "Grails Version: " + VersionInfo.getGrailsVersion(),
+                "Grails Java Version: " + JdkVersion.DEFAULT_OPTION.majorVersion(),
```

Comment:
@matrei  how about I change this to the same logic as in create command - where if defined it uses the system version, otherwise it will use Grails default JDK version?

Reply:
>I debated whether it would be helpful to include: The issue is the GraalVM java version will be one (almost always newer) value, while the system jdk will be another. Maybe for clarity we don't show the GraalVM version at all since it's internal to the cli? So we would show the grails version which will be initialized based on the system.java property or the default if not specified?

Yes, the GraalVM Java version is totally irrelevant to the user. The Grails version, and as a bonus, if we could show the currently used JVM on the system, but otherwise leave it out.

> how about I change this to the same logic as in create command - where if defined it uses the system version, otherwise it will use Grails default JDK version?

What do you mean by the "Grails default JDK version"? The baseline Java version? I'm don't know if that is helpful or confusing.

Reply:
Ok, I understand now.  Will make the change and circle back.